arch/arm64/src/common/arm64_vectors.S

```diff
@@ -139,164 +139,33 @@ SECTION_FUNC(text, arm64_jump_to_user)
 
 GTEXT(arm64_sync_exc)
 SECTION_FUNC(text, arm64_sync_exc)
-    /* checking the EC value to see which exception need to be handle */
-
-#if CONFIG_ARCH_ARM64_EXCEPTION_LEVEL == 3
-    mrs    x0, esr_el3
+    /* Switch to IRQ stack and save current sp on it. */
+#ifdef CONFIG_SMP
```

Comment:
@GUIDINGLI I assume this is the code that SMP needs ? I can adapt this part so that kernel mode works like before. I'll try to provide a patch tomorrow, or on Monday.

Reply:
good